.golangci.json

```diff
@@ -10,17 +10,34 @@
 			"gofmt"
 		]
 	},
+	"linters-settings": {
+		"govet": {
+			"check-shadowing": false
+		}
+	},
 	"run": {
 		"skip-files": [
 			"/zz_generated_"
 		],
+		"skip-dirs": [
```

Comment:
Does this have an option for "include" rather than "exclude"? Might end up being easier. No big deal either way.

Reply:
weirdly... no

Reply:
Fair enough. :(